src/app/components/Figure/Copyright/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+const copyrightSymbolPrefix = '\u00A9';
+
+const StyledCopyright = styled.span`
+  ${T_MINION};
+  background-color: ${C_EBON};
+  color: ${C_WHITE};
+  padding: ${GEL_SPACING};
+  font-family: ${FF_NEWS_SANS_REG};
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`;
+
+const Copyright = ({ text }) => (
```

Comment:
Can this be changed to `children` instead of text please. Therefore in the future if the copyright is a different lang to the rest of the document we can pass in the already handle HTML. EG: if this was an arabic company we may want to have the output `© <span lang='ar`>صاحب حقوق الطبع والنشر</span>